### ISSUE TITLE
Account card too big on Search All results

### ISSUE THREAD

**Issue by dhuangg** [omou-org/front-end#716]
# Steps to Recreate
1. Search for something under the "All" filter
2. View account results with at least 4 results. The 4th card is on the next line
<img width="1512" alt="Screen Shot 2021-02-02 at 11 27 38 AM" src="https://user-images.githubusercontent.com/12959959/106652050-c8c05d00-6549-11eb-8ba6-46cb665aac99.png">

# Development Todo
- [ ] Resize the account card or adjust styling to fit 4 cards on the same line

**Comment by CalvinFronda** [comment]
It looks like it does fit within 4 per row - Is the design to keep them 4 per row and resize the card when the screen size goes down or am i mistaken? 

<img width="1679" alt="Screen Shot 2021-02-10 at 11 28 32 AM" src="https://user-images.githubusercontent.com/40410706/107561213-1e70b700-6b93-11eb-8f80-e79f8dc20b41.png">

**Comment by dhuangg** [comment]
Yeah, the cards should be responsive so we will see 4 cards per row down to
an ipad size
